WordPress/Classes/System/3DTouch/WP3DTouchShortcutCreator.swift

```diff
@@ -26,11 +26,13 @@ public class WP3DTouchShortcutCreator: NSObject
         blogService = BlogService(managedObjectContext: mainContext)
     }
     
-    public func createShortcuts(loggedIn: Bool) {
-        if loggedIn {
-            createLoggedInShortcuts()
-        } else {
-            createLoggedOutShortcuts()
+    public func createShortcutsIf3DTouchAvailable(loggedIn: Bool) {
+        if is3DTouchAvailable() {
```

Comment:
Very nitpicky code style (sorry) ... maybe an early return instead of nested logic in the if block?